### ISSUE TITLE
repeater field error

### ISSUE THREAD

**Issue by kuzyk** [rainlab/translate-plugin#193]
sorry for my english

I try to explain issue

so there is a model with translatable fields ['label', 'options' ....]

options is repeater with 2 fields ['key','label']

when try update model plugin throw exception "htmlentities() expects parameter 1 to be string, array given" in this [file](https://github.com/rainlab/translate-plugin/blob/master/traits/mlcontrol/partials/_locale_values.htm#L9) because try to translate 'label' field of repeater

**Comment by daftspunk** [comment]
Make sure you specify `options` as "jsonable"

    /**
     * @var array List of attribute names which are json encoded and decoded from the database.
     */
    protected $jsonable = ['options'];

**Comment by CptMeatball** [comment]
I think this is not a question, but actually a bug. 

When a MLRepeater is used but not filled in, the framework throws an error.

**Expected behavior**

Empty MLRepeater shows an empty `repeater` form field.

Actual behavior

MLRepeater throws error:

 ``` htmlentities() expects parameter 1 to be string, array given ```

Reproduce steps

0. Install `Translate`.
1. Define a `repeater` type form field in `forms.yaml`.
2. Add extra repeater form field to `$jsonable` and `$translatable`
3. Add new Model
4. Do not fill in repeater form field.

October build

October **389** and Translate **1.2.8**

**Comment by H6** [comment]
Hi,

i have the same issue here.

The problem occurs in `plugins/rainlab/translate/traits/mlcontrol/partials/_locale_values.htm` at the point when the laravel `e` helper method wants to escape an `array` instead of a `string`.

I am not sure, what kind of value should be inserted into value in case of an array, but if the array is empty there could be added a check into `_locale_values.htm` which sets the value as an empty string..

```php
$value = ($value = $this->getLocaleValue($code)) && $this->isLocaleFieldJsonable() ? json_encode($value) : $value;
if (empty($value))
   $value = "";

```

**Comment by H6** [comment]
I have added a PL https://github.com/rainlab/translate-plugin/pull/210 ...

**Comment by LukeTowers** [comment]
Fixed by #210
